src/tools/base/BaseAnnotationTool.js

```diff
@@ -162,18 +155,6 @@ class BaseAnnotationTool extends BaseTool {
   toolSelectedCallback(evt, annotation, interactionType = 'mouse') {
     moveAnnotation(evt, this, annotation, interactionType);
   }
-
-  /**
-   * Updates cached statistics for the tool's annotation data on the element
-   *
-   * @param {*} image
-   * @param {*} element
-   * @param {*} data
-   * @returns {void}
-   */
-  updateCachedStats(image, element, data) {
-    logger.warn(`updateCachedStats not implemented for ${this.name}.`);
-  }
```

Comment:
Hmm... It says that it's abstract and must be implemented. Why was this nuked? At the very least, it would be virtual to indicate usage/pattern?

Reply:
So it looks like we added this to force annotation tools to consistently update their data in the same way. `moveHandles` verifies tool is an instance of `BaseAnnotationTool`, and it calls its underlying `updateCachedStats` method. I don't love the implementation, but removing this means we don't surface a helpful warning to tool authors.